client/js/components/snippets/snippet-form-dialog.jsx

```diff
@@ -31,16 +37,32 @@ export default class SnippetFormDialog extends React.Component {
       content: this.state.content,
       description: this.state.description,
       language: this.state.language
+    }).then(() => {
```

Comment:
@vrael560 I also think that we don't need to wait with opened form for a response. We should just reset form and close it. Only snackabr element should be displayed when snippet was properly saved in DB. But we can listen for store change.